### PR TITLE
[ Selenium ] Fix unexpected fail of "CheckWorkingWithTabsByUsingContextMenuTest" selenium test

### PR DESCRIPTION
**Pull request by Ohrimenko1988**
### What does this PR do?
Fix unexpected fail of "CheckWorkingWithTabsByUsingContextMenuTest" selenium test

### What issues does this PR fix or reference?
Issue: https://github.com/eclipse/che/issues/11276

### CODE REVIEW

**Review by artaleks9 — APPROVED** [review]

**Comment by artaleks9** [comment]
Selenium tests execution on Eclipse Che Multiuser on OCP (https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/828/) doesn't show any regression against this Pull Request.

**Review by musienko-maxim — APPROVED** [review]

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/editor/CheckWorkingWithTabsByUsingContextMenuTest.java [hunk 1]
```diff
@@ -96,16 +96,6 @@ public void checkWorkingWithTabsByUsingContextMenu() {
     editor.waitTabIsNotPresent(NAME_TO_HTML);
     editor.closeAllTabs();
 
-    // close other tabs by using context menu
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/editor/CheckWorkingWithTabsByUsingContextMenuTest.java:99]: Do you know what was the propose to repeat the same verification twice?

Comment (Ohrimenko1988) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/editor/CheckWorkingWithTabsByUsingContextMenuTest.java:99]: No, I don't, but it hasn't any explanation, so I have decided that this is duplicated code and removed it.

Comment (dmytro-ndp) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/editor/CheckWorkingWithTabsByUsingContextMenuTest.java:99]: Okay.

**Review by dmytro-ndp — APPROVED** [review]

**Pull request merged**